examples/AlfaKit/Sample/SampleTemperatureAndHumidity.h

```diff
@@ -16,7 +16,7 @@
 DHT dht(SENSOR_PIN, DHTTYPE);
 
 void sample_temperature_humidity_setup() {
-  pinMode(SENSOR_PIN, INPUT);
+  dht.begin();
```

Comment:
Functionally equivalent, but dht.begin() lets the DHT library do any other init it wants too.